book_source/03_topical_pages/03_pecan_xml.Rmd

```diff
@@ -636,6 +636,42 @@ This information is currently used by the following PEcAn workflow functions:
 - `PEcAn.<MODEL>::write.configs.<MODEL>` -- See [above](#pecan-write-configs)
 - `PEcAn.uncertainty::run.sensitivity.analysis` -- Executes the uncertainty analysis
 
+#### Coordinating inputs with the `input_design` design matrix {#xml-input-design}
+
+Multi-site ensembles and sensitivity analyses that sample over input files use
+an `input_design` data.frame to keep parameter draws and input files aligned
+across runs. The design is created up front (typically via
+`generate_joint_ensemble_design()` inside `runModule.run.write.configs()`) and
```

Comment:
I'd drop the "inside" bit. I think the preferred usage would be to generate the design first then pass it in.